packages/@okta/vuepress-site/docs/concepts/interaction-code/index.md

```diff
@@ -34,37 +34,12 @@ The following table describes the parameters introduced for the Interaction Code
 
 The following sequence of steps is a typical Interaction Code flow:
 
-<!--
```

Comment:
please add these all back - we want to keep them for future reference. Thanks.

Reply:
@susanharper-okta @vanngo-okta  Have reinstated all the plantuml code. Not sure how they are good for quick fixes though.